cartridges/sfra_demo/cartridges/sfra_demo/cartridge/controllers/Error.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var server = require('server');
+server.extend(module.superModule);
+
+server.append('Start', function (req, res, next) {
+
+    var viewData = res.getViewData();
+    viewData.demo1 = {
+        testVar: 'Demo1 cartridge appending to error page'
+    };
+    res.setViewData(viewData);
```

Comment:
No need to use getViewData. You can just simply put
res.setViewData({
    demo1: {
        testVar: '....'
     }
});